monai/networks/layers/simplelayers.py

```diff
@@ -41,38 +42,39 @@ def __init__(self, spatial_dims, sigma, truncated=4.0, device=None):
         Args:
             spatial_dims (int): number of spatial dimensions of the input image.
                 must have shape (Batch, channels, H[, W, ...]).
-            sigma (float): std.
+            sigma (float or sequence of floats): std.
             truncated (float): spreads how many stds.
             device (torch.device): device on which the tensor will be allocated.
         """
-        self.kernel = torch.nn.Parameter(torch.tensor(gaussian_1d(sigma, truncated)), False)
         self.spatial_dims = spatial_dims
-        self.conv_n = [F.conv1d, F.conv2d, F.conv3d][spatial_dims - 1]
-        self.padding = same_padding(self.kernel.size()[0])
+        _sigma = ensure_tuple_rep(sigma, spatial_dims)
         self.device = device
-
-        self.kernel = self.kernel.to(self.device)
+        self.kernel = [
+            torch.nn.Parameter(torch.from_numpy(gaussian_1d(s, truncated)).to(self.device), False) for s in _sigma
+        ]
+        self.padding = [same_padding(k.size()[0]) for k in self.kernel]
+        self.conv_n = [F.conv1d, F.conv2d, F.conv3d][spatial_dims - 1]
 
     def __call__(self, x):
         """
         Args:
             x (tensor): in shape [Batch, chns, H, W, D].
         """
         if not torch.is_tensor(x):
-            x = torch.Tensor(x)
+            x = torch.as_tensor(x)
```

Comment:
`torch.as_tensor()` will do nothing if the data is already a tensor, so I think we don't need `if not torch.is_tensor(x):`

Reply:
thanks!